netci.groovy

```diff
@@ -39,7 +39,7 @@ def branch = GithubBranchName
                     Utilities.setMachineAffinity(newJob, 'Windows_NT', 'latest-or-auto')
                     break;
 
-                case "Ubuntu16.04":
+                case "Ubuntu14.04":
                     // Define your build/test strings here.
                     def buildString = """./build.sh -c ${configuration}"""
```

Comment:
Not going to run tests?

Reply:
Not as part of this PR.

Reply:
I think dotnet/cli(dotnet/<any_repo>) runs tests with `build.cmd`, I think `source-build` expects tests should run as part of `build`.

Reply:
I am not sure, let's sync up. Here's the repo API [doc](https://github.com/dotnet/source-build/blob/dev/release/2.0/Documentation/RepoApi.md) I referred.